tools/js-sdk-release-tools/src/hlcCodeGenCli.ts

```diff
@@ -21,6 +21,8 @@ async function automationGenerateInTerminal(absoluteReadmeMd: string, tag?: stri
             tag: tag,
             use: use,
             additionalArgs: additionalArgs,
+            apiVersion: "",
```

Comment:
the HLC changelog repects `generateinput`, while codegen does not...
Could you make the option optional or suport Swagger?